components/ProjectDetail/ProjectDetail.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+const ProjectContainer = styled.div`
+  background-color: #faf8f7;
+  margin-left: 20px;
+  padding: 0 10px 10px 10px;
+  margin-bottom: 20px;
+  border-radius: 5%;
+`;
+
+const StyledImage = styled(Image)`
+  height: 100%;
+  width: 100%;
+  border-radius: 5%;
+`;
+
+export default function ProjectDetail({
+  project: {
+    category,
+    title,
+    imageSource,
+    width,
+    height,
+    longDescription,
+    organizer,
+    contact,
+  },
```

Comment:
I would recommend to distructure your prop in the function body:
```suggestion
  project
```

Reply:
Cool! Thank you!